### PR TITLE
Contractors Data

### PR DESCRIPTION
**Pull request by davidalenoghena**
Made the requested changes and now the information for the data is gotten from the database

### CODE REVIEW

File: database/factories/CompanyFactory.php [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Company;
+use Faker\Generator as Faker;
+
+$factory->define(Company::class, function (Faker $faker) {
+    return [
+        //
+        'snumber' => $faker->snumber,
+        'service' => $faker->service,
```
Comment (goodhands) [inline database/factories/CompanyFactory.php]: why are you changing the table name?

**Review by goodhands — CHANGES_REQUESTED** [review]

File: database/factories/CompanyFactory.php (hunk 1 above)
Comment (davidalenoghena) [reply database/factories/CompanyFactory.php]: I changed it with respect to the valid information I could get online

Comment (davidalenoghena) [reply database/factories/CompanyFactory.php]: I changed it to suit what we have best

Comment (davidalenoghena) [reply database/factories/CompanyFactory.php]: What changes do you need me to make before you merge this?

**Comment by davidalenoghena** [comment]
Now using the names of the current database. Sorry for the inconvenience

File: database/seeds/CompanySeeder.php [hunk 2]
```diff
@@ -1,61 +1,94 @@
...
+     *
+     * @return void
+     */
+    public function run()
+    {
+        //
+        DB::table('companies')->insert([
+        [
+            "name" =>"1",
+            "shortname" => "Revised Estimated Total Cost
```
Comment (goodhands) [inline database/seeds/CompanySeeder.php:19]: this is not a shortname

File: database/seeds/CompanySeeder.php [hunk 3]
```diff
@@ -1,61 +1,94 @@
...
+            for the Constructions of 
+            2X60MVA, 132/33KV 
+            Substation at Odogunyan, 
+            2X60MVA, 132/33KV 
+            Substation at Ayobo, Ikeja 
+            West-Ayobo 132KV D/C 
+            Transmission Lines and 
+            2X132KV Lines Bays Extension 
+            at Ikeja West",
+            "industry"=> "LAGA CE Power Limited",
```
Comment (goodhands) [inline database/seeds/CompanySeeder.php:29]: this is not an industry

File: database/seeds/CompanySeeder.php [hunk 4]
```diff
@@ -1,61 +1,94 @@
...
+            "ceo"=> "N3,502,347,285.84",
```
Comment (goodhands) [inline database/seeds/CompanySeeder.php:30]: this is not a person's name

**Review by goodhands — CHANGES_REQUESTED** [review]

**Comment by goodhands** [comment]
I will go ahead and close this PR now, I advice you pick up another task

**Pull request closed without merging**